src/pages/other-services/resources-and-toolkits/covid-19-kit-for-parents.tsx

```diff
@@ -0,0 +1,103 @@
+import Link from "next/link";
+
+import { List } from "@/components/Services/list";
+
+export default function ResourcesAndToolkits() {
+  const covidBookletList = (
+    <List
+      items={[
+        "families and organisations with practical tips, activities and links to resources that are focused on women and children and young people’s health and wellbeing.",
+        "offers guidance, support and information for women whose families are in isolation."
+      ]}
+    />
+  );
+
+  const body = (
+    <>
+      <div className="flex justify-center bg-[--starick-white] px-6 py-2 md:px-32">
+        <div className="w-fit">
+          <br></br>
+          <p>
+            Starick’s COVID-19 resource booklets were developed in consultation
+            with children, young people and families living in refuges or
+            receiving support from FDV services.{" "}
+          </p>
+          <br></br>
+          <p>The booklet provides</p>
+          {covidBookletList}
+          <br></br>
+          <p>
+            As we move into living with COVID-19, this booklet can be used more
+            broadly to support families and children living in refuges and
+            outreach services
+          </p>
+          <br></br>
+          <p>
+            The booklets provide direct links to online resources. Please make
+            sure that supervision of children and young people online is age
+            appropriate.
+          </p>
+          <br></br>
+          <br></br>
+        </div>
+      </div>
+    </>
+  );
+
+  return (
+    <>
+      <div>
+        <img
+          src="/images/Parents-with-kids-graphic.png"
+          className="h-[500px] w-full object-cover"
+          alt=""
+        ></img>
+        <div className="relative -mt-20 mb-6 w-fit rounded-r-full bg-white px-10">
+          <p className="p-2 text-4xl font-bold text-[--starick-brown]">
+            COVID-19 Kit for Parents
+          </p>
+        </div>
+      </div>
+      {body}
+
+      <div className="-mx-2 flex flex-wrap justify-center bg-[--starick-white]">
```

Comment:
It might be better to refactor this code to follow the DRY principle and have a json object with all the different languages, and loop through it here with a js map function, as [here](https://react.dev/reference/react/Children#children-map)